_posts/en/newsletters/2018-10-16-newsletter.md

```diff
@@ -0,0 +1,151 @@
+---
+title: "Bitcoin Optech Newsletter #17"
+permalink: /en/newsletters/2018/10/16/
+name: 2018-10-16-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter briefly describes a proposal for splicing
+Lightning Network payment channels, links to videos and transcripts of
+talks from the Edge Dev++ training sessions, and summarizes some
+transcripts made during last week's CoreDev.tech event.
+
+## Action items
+
+None this week.
+
+## News
+
+- **Proposal for Lightning Network payment channel splicing:** splicing
+  is an idea for allowing users to either add or remove funds from an
+  existing payment channel without the delay of closing and reopening a
+  completely new channel.  Rusty Russell posted a [technical
+  proposal][complex splice] for allowing a single splice at a time,
+  although he notes that the proposal is complex.   René Pickhardt
+  described an [alternative][simpler splice] that would likely be easier
+  to implement and reason about but which could require more onchain
+  transactions.  It was suggested that the simpler but more expensive
+  solution could be version 1, and the more complex but less expensive
+  solution could be version 2.
+
+- **Edge Dev++ talks published:** a two-day series of presentations from
+  leading Bitcoin contributors aimed at developers has been published
+  as [videos][dev vids] and [transcripts][dev transcripts].  The talks
+  cover the full range of topics, from introductory to advanced.  Three
+  talks may be especially interesting to Optech members:
+
+    1. *Exchange security* by Warren Togami.  Describes the causes
+       behind several notable major thefts from Bitcoin and altcoin
+       exchanges and lists a number of techniques businesses can use to
+       reduce their risk of loss.  ([Video][warren vid],
+       [transcript][warren transcript])
+
+    2. *Wallet Security, Key Management & Hardware Security Modules
+       (HSMs)]* by Bryan Bishop.  Suggests methods for decreasing the
+       risk that private keys will be stolen or misused.
+       ([Video][kanzure wallet vid], [transcript][kanzure wallet
+       transcript])
+
+    3. *Handling reorgs and forks* by Bryan Bishop.  Describes how to
+       secure your transactions against changes in the Bitcoin block
+       chain or consensus rules, including suggestions for how to test
+       your systems.  ([Video][kanzure reorg vid], no transcript)
```

Comment:
An anonymous donor has contributed a transcript for the reorgs talk: http://diyhpl.us/wiki/transcripts/scalingbitcoin/tokyo-2018/edgedevplusplus/reorgs/

Reply:
Thanks @kanzure! PR here: https://github.com/bitcoinops/bitcoinops.github.io/pull/77